packages/wonder-blocks-modal/src/components/modal-launcher.tsx

```diff
@@ -153,16 +157,12 @@ const ModalLauncher = (props: Props): React.ReactElement | null => {
     const returnFocus = React.useCallback(() => {
         // Focus on the specified element after closing the modal.
         if (closedFocusId) {
-            // eslint-disable-next-line import/no-deprecated
-            const focusElement = ReactDOM.findDOMNode(
-                document.getElementById(closedFocusId),
-            ) as any;
-
+            const focusElement = document.getElementById(closedFocusId);
             if (focusElement) {
-                // Wait for the modal to leave the DOM before trying
-                // to focus on the specified element.
-                schedule.animationFrame(() => {
-                    focusElement.focus();
+                // Use the scheduler to ensure testability
+                schedule.animationFrame(() => focusElement.focus(), {
+                    schedulePolicy: SchedulePolicy.Immediately,
+                    clearPolicy: ClearPolicy.Resolve,
```

Comment:
Interesting, I haven't worked with our timing utilities yet, so this is good to know!

Looking into it, the default `schedulePolicy` for [animationFrame](https://khan.github.io/wonder-blocks/?path=/docs/packages-timing-types-ischeduleactions--docs#animationframe) is `ClearPolicy.Cancel`. Does that mean setting it to `ClearPolicy.Resolve` is what addresses the bug?

Reply:
Apparently so! The `clearPolicy` change is the critical line to making the scheduler fire for some reason. Without it, the code fails silently. The `schedulePolicy` doesn't seem to have an effect.